server/app/routes/api/v1/users/spec.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
This seems like it should be yet another PR -- one for testing potentially. I'm not going to go over this one -- I'm just going to finish what looks like the PR for routes. Next time just 1 PR guys 😺